docs/node/1-node-setup.md

```diff
@@ -0,0 +1,113 @@
+# 📦 AtomOne Node Setup Guide
+> Replace YOUR_MONIKER_GOES_HERE with your validator name
+
+```
+MONIKER="YOUR_MONIKER_GOES_HERE"
+```
+
+## Update system and install build tools
+First, let's make sure your system is up-to-date and install the required build tools. These are essential for compiling the binaries for AtomOne.
+
+```
+sudo apt -q update
+sudo apt -qy install curl git jq lz4 build-essential
+sudo apt -qy upgrade
+```
+## Install Go
+AtomOne requires Go for building the binaries. We will install the Go programming language and set the appropriate environment variables.
+```
+sudo rm -rf /usr/local/go
+curl -Ls https://go.dev/dl/go1.22.10.linux-amd64.tar.gz | sudo tar -xzf - -C /usr/local
+eval $(echo 'export PATH=$PATH:/usr/local/go/bin' | sudo tee /etc/profile.d/golang.sh)
+eval $(echo 'export PATH=$PATH:$HOME/go/bin' | tee -a $HOME/.profile)
+```
+## Download and build binaries
+Next, we will clone the AtomOne project repository, check out the specific version, and build the binaries required for running the node.
+```
+# Clone project repository
+cd $HOME
+rm -rf atomone
+git clone https://github.com/atomone-hub/atomone
+cd atomone
+git checkout v1.1.2
```

Comment:
What do you think about using `$VERSION` rather than v1.1.2 so we don't have to update the latest version everytime there is one ?

cc: @Stuyk

Reply:
I think that's fine. I'm indifferent on the subject matter here.